### PR TITLE
feat: Download invoices as pdf

### PR DESCRIPTION
**Pull request by enguerran**
It downloads pdf :yatta:

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -32,42 +34,96 @@ async function start(fields) {
...
+      'content-type': 'application/json;charset=UTF-8'
+    }),
+    validate: response => response && JSON.parse(response.body).user.id,
+    debug: false
+  }
+
+  return await signin(options)
+}
+
+async function signin({
```
Comment (enguerran) [inline src/index.js]: this is a brand new `signin` function.

**Comment by enguerran** [comment]
@konnectors/maintainers travis will remain unhappy until I make the packaging.

But this PR is ready to review.

File: src/index.js [hunk 2]
```diff
@@ -1,122 +1,157 @@
...
+      identifiers: ['deliveroo']
     }
+  )
+}
+
+async function authenticate(username, password) {
+  const options = {
+    formselector: 'form#login',
+    input: {
+      login: 'login_email',
```
Comment (doubleface) [inline src/index.js]: This is weird : login: 'login_email'

File: src/index.js [hunk 3]
```diff
@@ -1,122 +1,157 @@
...
+  const loginpage = await request({
+    uri: loginurl,
+    resolveWithFullResponse: true,
+    transform: (body, response, resolveWithFullResponse) =>
+      resolveWithFullResponse
+        ? { ...response, parsedBody: cheerio.load(body) }
+        : cheerio.load(body),
+    jar: cookiejar
+  }).catch(err => {
+    console.error(err)
```
Comment (doubleface) [inline src/index.js]: Be carefull, these errors are not shown in the logs in production. This should be a : 

```javascript
log('error', err)
```

File: src/index.js [hunk 4]
```diff
@@ -1,122 +1,157 @@
...
+  const headers = generateHeader($)
+  const response = await request({
+    method: 'POST',
+    uri: formurl,
+    form,
+    headers,
+    resolveWithFullResponse: true,
+    jar: cookiejar
+  }).catch(err => {
+    console.error(err)
```
Comment (doubleface) [inline src/index.js]: Same here

**Comment by doubleface** [comment]
LGTM: Just some tiny changes about error logs.

File: src/index.js (hunk 2 above)
Comment (enguerran) [reply src/index.js]: It means that the login input is named `login_email`.
According to the classic form of a web signin form: login / password.

Comment (enguerran) [reply src/index.js]: I named the configuration object for input fields:
```js
const input = {
  input_login_name: "login_email",
  input_password_name: "login_password"
}
```

Do you find it less weird?

**Pull request merged**